_includes/releases/v22.2/v22.2.0-beta.2.md

```diff
@@ -0,0 +1,91 @@
+
+<h3 id="v22-2-0-beta-2-db-console-changes">DB Console changes</h3>
+
+- Removed the "full scan" field from the **Active Transaction Details** page. [#88404][#88404]
+- Added support for multiple blocking transactions on the **Insights** > **Transaction Executions** page, merged displayed cards into the table, and fixed the reported total contention time. [#88522][#88522]
+- Renamed the **Insights Overview** table column "execution id" to "latest execution id". This will help avoid confusion since the UI only shows the latest ID per fingerprint. [#88591][#88591]
+- The {{ site.data.products.dedicated }} Cloud Console now displays information on which transactions are blocked or waiting on the active execution pages. This includes the `Time Spent Waiting` column in the overview pages and the wait time insights panel on the details pages. This feature is currently unavailable for {{ site.data.products.serverless }}. [#88856][#88856]
+- The **Statements**, **Transactions**, and **Transaction Details** pages in the DB Console now properly show the Regions/Nodes columns, as well as filters for multi-region clusters. [#88572][#88572]
+
+<h3 id="v22-2-0-beta-2-bug-fixes">Bug fixes</h3>
+
+- CockroachDB no longer fetches unnecessary rows for queries that use the [`LIMIT`](../{{site.versions["dev"]}}/limit-offset.html) statement. The bug was introduced in v22.1.7. [#88417][#88417]
+- Active transactions with a non-zero executed statement count now always have populated statement text, even when no statement is being executed. [#88404][#88404]
+- Fixed a bug where offline tables were included in [full-cluster backups](../{{site.versions["dev"]}}/backup.html#backup-a-cluster), as reported in [#88043](https://github.com/cockroachdb/cockroach/issues/88043). [#88474][#88474]
+- Implemented a change to ensure that time elapsed for active transactions and statements is never negative. [#88467][#88467]
+- Fixed a rare internal error with message `estimated row count must be non-zero`, which could occur during planning when a predicate included values close to the maximum or minimum `int64` value. [#88529][#88529]
+- Upgraded `grpc` to v1.49.0 which fixed a few panics on nil pointers [#88630][#88630]
+- Fixed missing automatic statistics collection on system tables during cluster startup. This bug only affects the v22.2.0-alpha releases. [#88689][#88689]
+- A bug has been fixed that could cause nodes to crash in rare cases when executing [apply joins](../{{site.versions["dev"]}}/joins.html#apply-joins) in query plans. [#88483][#88483]
+- A bug has been fixed that caused errors in rare cases when executing queries with correlated [`WITH`](../{{site.versions["dev"]}}/common-table-expressions.html) expressions. This bug has been present since the introduction of correlated `WITH` expressions in v21.2.0. [#88483][#88483]
```

Comment:
```suggestion
- Fixed a bug that caused errors in rare cases when executing queries with correlated [`WITH`](../{{site.versions["dev"]}}/common-table-expressions.html) expressions. This bug has been present since the introduction of correlated `WITH` expressions in v21.2.0. [#88483][#88483]
```